float32 yellow_location
float32 black_location
float32 red_location
float32 green_location
float32 middle
bool isyellowfound
bool isredfound
bool isgreenfound
bool isblackfound

